src/main/java/seedu/address/model/employee/Employee.java

```diff
@@ -13,16 +13,18 @@
  */
 public class Employee {
 
+    private EmployeePay employeePay;
```

Comment:
EmployeePay would refer to the Pay/hr of the Employee (can leave it as it is).
However, `EmployeeTotalSalary` and `EmployeePendingPay` objects should be put under a container class called perhaps `EmployeeFinances`, due to how often it mutates. They **should not be a field** on its own. This object is not included at all in AddEmployee/EditEmployee commands, and is accessed/edited through other commands. You may look at `Event.java` and `EventDateTimeMap.java` to see how this is implemented.